docker/ray-ml/Dockerfile

```diff
@@ -46,6 +51,14 @@ RUN sudo apt-get update \
         requirements*.txt \
     && sudo apt-get clean
 
+# MuJoCo Installation.
+ENV MUJOCO_GL=osmesa
```

Comment:
if you use `v4` envs, you should be able to download the new bindings via the `mujoco` python package, it is much easier to install than `mujoco-py` + manual mujoco download.

Reply:
to make it clear: `mujoco` python package doesn't require a manual install of mujoco anymore

Reply:
I did try that (replacing mujoco-py wih mujoco), but it still gave me a "MoJoCo not installed" error. So I reverted back to what I knew and used mujoco-py plus the tar download and extract procedure.

Reply:
Thanks for the hint @araffin ! We can do this in a follow up PR now that all tests are green (just want to keep it this way :) ).